app.rb

```diff
@@ -0,0 +1,99 @@
+require './person'
+require './classroom'
+require './book'
+require './student'
+require './rental'
+require './teacher'
+
+class App
+  def initialize
+    @classroom = Classroom.new('Unknown')
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "#{person.role} - #{person.name}" }
```

Comment:
The console app is required to function like [the sample here](https://www.youtube.com/watch?v=vkkgrhD6aXQ)

You should list other attributes of the person to help test this.  Such as the `id`. Recall, that the ID is needed to list the rentals.

This could not be pointed out by the previous reviewer because the error broke the code before they got to this point.